.changeset/selfish-numbers-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `selector-class-pattern` end positions
```

Comment:
[suggest] Minor. The following seems more accurate:

```suggestion
Fixed: `selector-class-pattern` reported ranges
```

https://github.com/stylelint/stylelint/blob/f1fd5748f9c926b1c5360df5f5e874b22a65357f/CHANGELOG.md?plain=1#L8

Reply:
`index` didn't change but fine by me.